script/Common/00_00.txt

```diff
@@ -865,7 +865,7 @@ I can't go anywhere. I %{i}shouldn't%{/i} go anywhere.
 [sha:1dce5e8d5e312ad6d9ae45dc5e1a006be86ea28e]{
 -- Page 57, Offset 271.
 -- ツギハギが見える人間は、きっとドコにも居てはいけない。
-A person who can see these lines doesn't belong anywhere.
+Someone who can see these lines don't belong anywhere.
```

Comment:
doesn't still correct

Reply:
Yes. I think I had it as 'people' briefly, and forgot.